views/javascript.js

```diff
@@ -33,8 +33,9 @@ var currentValue = {
 
 var side = { side: 'left' }
```

Comment:
This naming is confusing to me. Making this an object is not necessary if there is only one value. Replace with:
```js
var side = 'left'
```